ask-question.js

```diff
@@ -6,7 +6,8 @@ let controller = Botkit.slackbot({})
 let bot = controller.spawn()
 
 bot.configureIncomingWebhook({ url: config('WEBHOOK_URL') })
-const questions = require("./questions.json")
+const questions = require("./questions.js")
+questions.filter(q => q.hasBeenAsked)
```

Comment:
when do we mark the question as being asked?

Reply:
First, we learn how to write to a file in express. Two, we learn how to run this locally.

Reply:
Wait, wouldn't this just reset every time new questions are deployed. I'm dumb.

Reply:
updated!

Reply:
oooh. you're right -- we need to write it back to the file, eh?

Reply:
you'd need to commit it
its going to be a pain

Reply:
I know this sounds dumb, but we should use a DB

Reply:
Yeah, even if you write it to the file on the server it'll still need to come back to this repo.

Reply:
Would be easy to write an import script into a mongo db somewhere then pull items via mongoose and write to the instance afterwards

Reply:
I guess we could just not add any more questions until these are all done...

Reply:
> Would be easy to write an import script into a mongo db somewhere then pull items via mongoose and write to the instance afterwards

That sounds like _work_